scripts/toaster_configuration

```diff
@@ -0,0 +1,153 @@
+#!/bin/sh
+#
+# Copyright 2015-2016 Mentor Graphics Corporation
+#
+# This file is licensed under the terms of the GNU General Public License
+# version 2.  This program  is licensed "as is" without any warranty of any
+# kind, whether express or implied.
+
+echo "Creating toaster configuration"
+
+BUILDDIR=$*
+
+BBLAYERS=
+configured_layers () {
+   tac "$BUILDDIR/conf/bblayers.conf" | \
+   sed -n -e '/^"/,/^BBLAYERS = /{ /^BBLAYERS =/d; /^"/d; p;}' | \
+   awk {'print $1'}
+}
+
+configured_layers | while read layer; do
+   if [ -d "$layer" ]; then
+       #Once testing is done, remove this if.
+       if [ -d "$layer/.git" ]; then
+            rm -fr "$layer/.git"
+       fi
```

Comment:
Am I missing something, or will this just wipe the user's existing git repositories? If this isn't safe to run on setups with existing git repositories, there's no way I'm going to merge it.